migrations/20240320074436_create_index_owner_not_burned_cw721_token.ts

```diff
@@ -0,0 +1,11 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(
+    `CREATE INDEX IF NOT EXISTS cw721_token_owner_not_burned ON cw721_token USING btree (owner ASC NULLS LAST, burned ASC NULLS LAST) WHERE NOT burned;`
```

Comment:
có where not burned rồi thì btree owner là đc mà nhỉ?

Reply:
fixed